stdlib/testing/usage/storage.flux

```diff
@@ -0,0 +1,2210 @@
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:20:40Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:20:50Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:21:00Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:21:10Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:21:20Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:21:30Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:21:40Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:21:50Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:22:00Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:22:10Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:22:20Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:22:30Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:22:40Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:22:50Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:23:00Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:23:10Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:23:20Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:23:30Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:23:40Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:23:50Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:24:00Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:24:10Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a85ca099000
+,,0,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:24:20Z,34,gauge,storage_usage_org_bytes,storage-3,038b7a
```

Comment:
This query also does not filter on `_field`.  Seems like if another field was ever added, it would either produce a wrong answer or errors.